### PR TITLE
Removing Operator UI

### PR DESCRIPTION
**Pull request by dvaldivia**

### CODE REVIEW

**Comment by allanrogerr** [comment]
Reviewing

**Comment by allanrogerr** [comment]
@dvaldivia Im facing this when building the console image and using it on kind. Is this expected? Also where I was challenged for a token before, Im now being asked for a user/password. Is this also expected?
image

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -1,3 +1,72 @@
+# Building with MinIO
+
+To test console in it's shipping format, you need to build it from the MinIO repository, the following step will guide
```
Comment (allanrogerr) [inline DEVELOPMENT.md:3]:
```suggestion
To test console in its shipping format, you need to build it from the MinIO repository, the following step will guide
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -1,3 +1,72 @@
...
+you to do that.
+
+### 0. Building with UI Changes
+
+If you are performing changes in the UI components of console and want to test inside the MinIO binary, you need to
+build assets first.
+
+On the console folder run
```
Comment (allanrogerr) [inline DEVELOPMENT.md:11]:
```suggestion
In the console folder run
```

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -1,3 +1,72 @@
...
+
+Still in the MinIO folder, run
+
+```shell
+make build
+```
+
+# Testing on Kubernetes
+
+If you want to test console on kubernets, you can perform all the steps from `Building with MinIO`, but change `Step 3`
```
Comment (allanrogerr) [inline DEVELOPMENT.md:53]:
```suggestion
If you want to test console on kubernetes, you can perform all the steps from `Building with MinIO`, but change `Step 3`
```

**Review by allanrogerr — COMMENTED** [review]
Some spelling

**Comment by dvaldivia** [comment]
@allanrogerr all suggestions were merged

**Review by allanrogerr — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by jinapurapu — APPROVED** [review]

**Pull request merged**